portfolio/src/main/webapp/index.html

```diff
@@ -95,8 +95,8 @@ <h2>Projects</h2>
                 </tr>
                 <tr>
                     <td>
-                        •	Conceptualized the characters and environment of the game using Maya Autodesk and Photoshop 
-                        •	Implemented augmented reality to enhance the game by 3D mapping and having physical space awareness using the Google Tango platform 
+                        <p> •	Conceptualized the characters and environment of the game using Maya Autodesk and Photoshop </p>
```

Comment:
So, to make a bulleted list, you can use the `ul` or `ol` tags to define the list and the `li` tag for each list item. The two list types are unordered (ul) and ordered(ol). In this case, I'd recommend an unordered list.

What you want to do here is something like this:

```
<ul>
  <li>My first list item.</li>
  <li>My second list item.</li>
</ul>
```

Reply:
Note you don't have to put the bullets in yourself in this case, the browser is smart enough to bullet an unordered list. You can also refer to the `list-style` properties in CSS if you want further customization or styling options.

Reply:
Ok I will try this!